### PR TITLE
Suggest Hackathon details

### PR DESCRIPTION
**Pull request by mkitti**
## Checklist for merge
- [ ] I built the website locally and tested it using `Franklin.serve()`
- [ ] All CI checks have passed and you have tested the online version (click on the list of checks and click `Build and Deploy / Preview` to see the preview)
- [ ] Somebody else reviewed my changes (use your best judgement if you need to merge quickly

## After merge
- [ ] Closed relevant issues

### CODE REVIEW

**Review by MaxxxvD — APPROVED** [review]

File: 2024/hackathon.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
\ No newline at end of file
+This year the hackathon will take place in the iconic [Grand Cafe De Lichttoren](https://maps.app.goo.gl/wA2yHBx1Xp8LnqTG6) on Saturday, July 13th. Here you can chat with the friends you made during the previous days of JuliaCon, while in the meantime solving problem of package developers. The following projects are ready to be worked on:
+
+* Implement and test compression for interoperability between JLD2.jl, JLD2.jl, HDF5.jl, and Zarr.jl (Mark Kittisopikul)
+* Make Zarr.jl compatible with Zarr version 3 or create a Zarr3.jl (Mark Kittisopikul)
+* Create a tarball or zip archive format to aid in Julia package distribution (Mark Kittisopikul)
+* Create extension packages to integrate Images.jl with Makie.jl (Mark Kittisopikul)
+* Update Napari.jl to work with PythonCall.jl, integrate IJulia into the Napari console (Mark Kittisopikul)
+* Create an ImageJ.jl package via JavaCall.jl for ImageJ and Julia interoperation. (Mark Kittisopikul)
+* Add your own idea here. Suggest new projects via [Discourse](https://discourse.julialang.org/c/juliacon/juliacon2024/67) or Slack.
```
Comment (giordano) [inline 2024/hackathon.md:11]: Last years the topics were suggested in a collaborative hackmd document, which looks easier to update than a long thread on discourse, or relying on someone to update this list on github

Comment (mkitti) [reply 2024/hackathon.md:11]: Go ahead and start one then please. I'll create a link.

Comment (giordano) [reply 2024/hackathon.md:11]: [JuliaCon '24 Hackathon Ideas](https://hackmd.io/@mose/Hym4DRCIA/edit) (for the record, [this](https://hackmd.io/@mose/HkXioYGi3/edit) is the document we used last year)

Comment (mkitti) [inline 2024/hackathon.md:11]:
```suggestion
* Add your own idea here. Suggest new projects via [HackMD](https://hackmd.io/@mose/Hym4DRCIA/edit), [Discourse](https://discourse.julialang.org/c/juliacon/juliacon2024/67) or Slack.
```

**Pull request merged**